indico/modules/search/client/js/index.jsx

```diff
@@ -13,9 +13,12 @@ import SearchApp from './components/SearchApp';
 
 document.addEventListener('DOMContentLoaded', () => {
   const root = document.querySelector('#search-root');
+  const id = parseInt(root.dataset.id, 10) || undefined;
+  const title = root.dataset.title;
```

Comment:
`const category = root.dataset.category ? JSON.parse(root.dataset.category) : null;`

and in the template put something like `data-category="{{ {'id': category.id, 'title': category.title} | tojson | forceescape }}"`

Reply:
I've made this same change to `SearchBox` and the `Category` base template.